x-pack/plugins/security_solution/server/lib/telemetry/tasks/prebuilt_rule_alerts.ts

```diff
@@ -98,13 +98,19 @@ export function createTelemetryPrebuiltRuleAlertsTaskConfig(maxTelemetryBatch: n
 
           tlog(logger, `sending ${enrichedAlerts.length} elastic prebuilt alerts`);
           const batches = batchTelemetryRecords(enrichedAlerts, maxTelemetryBatch);
-          for (const batch of batches) {
-            await sender.sendOnDemand(TELEMETRY_CHANNEL_DETECTION_ALERTS, batch);
+
+          const promises = batches.map(async (batch) => {
+            sender.sendOnDemand(TELEMETRY_CHANNEL_DETECTION_ALERTS, batch);
+          });
+
+          try {
```

Comment:
do we need this try block?
We already have one on L115, if the purpose of this one is to close the pit, perhaps you could move it to a `finally` block, WDYT?

Reply:
Thanks! Updated in https://github.com/elastic/kibana/pull/170334/commits/94f8cea70322967c5f03e153dfffbde1328f88a2